k8s/postgresql/README.md

```diff
@@ -155,6 +158,25 @@ export POSTGRESQL_DB_PASSWORD=$(openssl rand 9 | openssl base64 -A | openssl bas
 export POSTGRESQL_VOLUME_SIZE=10
 ```
 
+##### Create dedicated service accounts
+
+Define the service accounts variable:
+
+```shell
+export POSTGRESQL_SERVICE_ACCOUNT="${APP_INSTANCE_NAME}-postgresql-sa"
+```
+
+Create the service account:
+
+```shell
+kubectl apply -f - <<EOF
```

Comment:
Do we care about namespace of this ServiceAccount?

Reply:
```suggestion
kubectl create serviceaccount ${POSTGRESQL_SERVICE_ACCOUNT} --namespace ${NAMESPACE}
```